### PR TITLE
Add technical part

### PR DESCRIPTION
**Pull request by kathoef**

### CODE REVIEW

File: 02_technology.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# What is a container?
+
+## High-level conceptual perspective
+
+* isolated runtime environment for “applications” w/ system dependencies, but w/o the kernel (unlike VMs!)
```
Comment (martinclaus) [inline 02_technology.md]: I would reformulate to "isolated runtime environment for “applications” including their dependencies but not, unlike VMs, the kernel."

File: 02_technology.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+* user processes
+* Linux kernel w/ file system
+* hardware layer
+
+## Hands-on part
+
+Let's pull the `alpine:latest` and `Ubuntu:21.04` Linux base images to illustrate the above concepts.
+
+On a Linux machine you can directly have a look at your host system's kernel version,
```
Comment (martinclaus) [inline 02_technology.md]: "Linux or OSX"

File: 02_technology.md (hunk 1 above)
Comment (kathoef) [reply 02_technology.md]: Thanks, changed that.

File: 02_technology.md (hunk 2 above)
Comment (kathoef) [reply 02_technology.md]: Also changed that. :)

**Comment by kathoef** [comment]
This is not fully ready yet, but at least has the necessary aspects. Merging for now.

**Pull request merged**